cmd/payledger/payledger.go

```diff
@@ -238,7 +258,7 @@ func savePayoutsFile(params *params.Params, genesisLedger map[string]*ledger.Tok
 	fileContent := fmt.Sprintf("package ledger\nfunc init%s() {\n", funName)
 
 	for k, v := range genesisLedger {
-		fileContent += fmt.Sprintf("	addPayout(\"%s\",%d,\"%s\")\n", k, v.Amount, hex.EncodeToString(v.PkScript))
+		fileContent += fmt.Sprintf("	addPayout(\"%s\",%d,\"%s\")\n", k, v.Payout.Amount, hex.EncodeToString(v.Payout.PkScript))
```

Comment:
fileContent += fmt.Sprintf("	addPayout(\"%s\",%d,\"%s\")\n", k, v.Payout.Amount+v.GenAmount, hex.EncodeToString(v.Payout.PkScript))

Reply:
The save-file part has not implemented properly yet, because we don't' want to do a  fresh replacement of the old one, instead, better to append it. so that the 0.8.3/0.8.4 testnet genesis rewards remain accountable.

Reply:
> The save-file part has not implemented properly yet, because we don't' want to do a fresh replacement of the old one, instead, better to append it. so that the 0.8.3/0.8.4 testnet genesis rewards remain accountable.

If 'v' is the UTXO  from genesis, v.Payout.Amount will equals zero.I mean, why not add v.GenAmount.

Reply:
not go into the detail of the save file logic yet, it's not running anyway by default. we can do further refactoring later.

Reply:
OK